README.md

```diff
@@ -59,7 +59,7 @@ To view the generated example project, just pull down the code and run ignite-ba
 Make sure you're set up for [React Native](https://facebook.github.io/react-native/docs/getting-started.html#content)
 
 **Step 1: Install**
-
+`npm start`
```

Comment:
you would want to start packager before you do `react-native run-ios` or `react-native run-android`.

So maybe down on line 81, you could say "If packager does not automatically start, you'll need to run `npm-start`"   That would be more helpful to others.